### PR TITLE
Display network status

### LINKED ISSUE #75: Display network status
**Description**:

Display network status. Google Docs is expressed as follows:

Offline
Screen Shot 2021-03-22 at 21 25 39

Online, synced
Screen Shot 2021-03-22 at 21 26 22

**Why**:

Editing works normally even when offline, but the changes are not saved on the server. We need to display to users that it is offline and that the network will need to be recovered before it will sync.

### PR DESCRIPTION
**Pull request by ppeeou**
#### What this PR does / why we need it?

Display network status to the user 
users can know that they can save a document even when offline.

#### What are the relevant tickets?

Fixes #75 

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: src/components/NavBar/NetworkButton.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'left',
+        }}
+      >
+        <NetworkAlert
+          title="All Changes saved to Server"
```
Comment (hackerwins) [inline src/components/NavBar/NetworkButton.tsx:84]: If changes couldn't be sent to the server, the message must be represented appropriately. We can check if any local changes couldn't be sent using `doc.hasLocalChanges()`.

No local changes:
```typescript
          title="All Changes saved to Server"
```

Has local changes:
```typescript
          title="There are changes that could not be sent to the server"
```

**Review by hackerwins — CHANGES_REQUESTED** [review]
Thanks for the contribution. I left a brief comment.

File: src/components/NavBar/NetworkButton.tsx (hunk 1 above)
Comment (ppeeou) [reply src/components/NavBar/NetworkButton.tsx]: @hackerwins Thanks for the comment. You are right!

**Review by hackerwins — APPROVED** [review]
LGTM. 👍

**Pull request merged**